src/en/disabilities/index.html

```diff
@@ -0,0 +1,32 @@
+---
+title: Most Common Types of Disability
+layout: layouts/base.njk
+description: Here are some of the most commonly found types of disabilities in the workplace, and their related accessibility considerations.
+---
+
+<div class="row wb-eqht">
+	<section class="col-md-6">
+		<h2 class="h3"><a href="./low-vision/">Low Vision</a></h2>
```

Comment:
It would probably be beneficial to leverage the doormat style. Like with the Service and information component. See: https://wet-boew.github.io/GCWeb/components/gc-servinfo/gc-srvinfo-examples.html